client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -128,6 +129,14 @@ const OnboardingEdit: React.FunctionComponent< BlockEditProps< Attributes > > =
 				</Route>
 
 				<Route path={ makePath( Step.CreateSite ) }>{ createSiteOrError() }</Route>
+
+				<Route path={ makePath( Step.FocusedLaunchFlowModal ) }>
+					<FocusedLaunchModal
```

Comment:
Can we async load this only when some feature flag is enabled? At the moment it's adding 21Kb gzipped to `/new` entry-point.
Also, when the same feature flag is disabled we should probably return `<Redirect to={ makePath(Step.AcquireIntent) } />`

Reply:
All good points, I'll make sure to add them to this PR

Reply:
- It no longer loads through gutenboarding's route, now load through calypso block editor route.
- It is now async loaded.